deps.edn

```diff
@@ -11,14 +11,21 @@
            org.clojure/clojure        {:mvn/version "1.10.1"}}
 
  :aliases {:test      {:extra-paths ["src/test"]
-                       :extra-deps  {fulcrologic/fulcro-spec {:mvn/version "3.1.5"}}}
+                       :extra-deps  {fulcrologic/fulcro-spec {:mvn/version "3.1.5"}
+                                     dev-local-tu            {:mvn/version "0.1.0"}
+
+                                     ; N.B. - follow instuctions for installing dev-local
+                                     ; https://docs.datomic.com/cloud/dev-local.html#using
+                                     com.datomic/dev-local {:mvn/version "0.9.184"}}}
```

Comment:
If there are both on-prem and cloud dev local tests, I think we should consider two diff test aliases with two diff source paths and deps. I have not read your tests, so let me know what you think